### PR TITLE
native: visual fix omnibus

### PR DESCRIPTION
**Pull request by jamesacklin**
- Updates the copy of WelcomeSheet (fixes TLON-2164)
- Brings gallery index view up to spec (fixes TLON-2251)
- Brings gallery detail views up to spec (fixes TLON-2161)
- Brings notebook index and detail views up to spec (fixes TLON-2162)

Does this by way of:

- Adding a check to usePostMeta() to determine if the text post is a non-image URL
- Adding a shortened text-only return to ContentRenderer to show preview text of a Notebook post in the channel
- Adding day suffixes to pretty dates (st, nd, rd, th)

### CODE REVIEW

File: packages/shared/src/logic/utils.ts [hunk 1]
```diff
@@ -98,9 +99,29 @@ export function makePrettyDay(date: Date) {
...
+    case 1:
+      return 'st';
+    case 2:
+      return 'nd';
+    case 3:
+      return 'rd';
+    default:
+      return 'th';
+  }
+}
```
Comment (patosullivan) [inline packages/shared/src/logic/utils.ts:124]: Why are we using this function? Doesn't `do` give us this? Is there some corner case we need covered that date-fns doesn't cover?

File: packages/ui/src/components/ChatMessage/ChatMessageReplySummary.tsx [hunk 2]
```diff
@@ -10,9 +10,11 @@ export const ChatMessageReplySummary = React.memo(
   function ChatMessageReplySummary({
     post,
     onPress,
+    paddingLeft = true,
   }: {
     post: db.Post;
     onPress?: () => void;
+    paddingLeft?: boolean;
```
Comment (patosullivan) [inline packages/ui/src/components/ChatMessage/ChatMessageReplySummary.tsx:17]: Might be good to leave a comment here explaining why this is necessary.

File: packages/ui/src/components/ContentReference/ChatReference.tsx [hunk 3]
```diff
@@ -44,6 +44,7 @@ export default function ChatReference({
           <ContactName
             color="$tertiaryText"
             size="$s"
+            maxWidth={230}
```
Comment (patosullivan) [inline packages/ui/src/components/ContentReference/ChatReference.tsx:47]: Would probably be good to leave a comment explaining why this specific number is used.

File: packages/ui/src/components/ContentReference/GroupReference.tsx [hunk 4]
```diff
@@ -24,14 +31,15 @@ export function GroupReference({ groupId }: { groupId: string }) {
...
-    <Reference onPress={onPress}>
+    <Reference onPress={onPress} viewMode={viewMode}>
       <Reference.Header>
         <Reference.Title>
           <ContactAvatar contactId={host} size="$xl" />
           <ContactName
             color="$tertiaryText"
             size="$s"
             userId={host}
+            maxWidth={230}
```
Comment (patosullivan) [inline packages/ui/src/components/ContentReference/GroupReference.tsx:42]: Same here

Comment (patosullivan) [reply packages/ui/src/components/ContentReference/GroupReference.tsx:42]: And if this is going to be used in a few places, maybe we define it in one place and import it where we need it.

File: packages/ui/src/components/GalleryPost/GalleryPost.tsx [hunk 5]
```diff
@@ -64,11 +70,13 @@ export default function GalleryPost({
...
       ? 'image'
-      : isText && !isLinkedImage && !isRefInText
+      : isText && !isLinkedImage && !isRefInText && !isLink
         ? 'text'
         : isReference || isRefInText
           ? 'reference'
-          : 'unsupported';
+          : isLink
+            ? 'link'
+            : 'unsupported';
```
Comment (patosullivan) [inline packages/ui/src/components/GalleryPost/GalleryPost.tsx:79]: Would probably be good to rewrite this without the ternaries now that it's gotten so complex.

File: packages/ui/src/components/NotebookPost/NotebookPost.tsx [hunk 6]
```diff
@@ -67,15 +64,14 @@ export default function NotebookPost({
             }}
             width="100%"
             height={smallImage ? IMAGE_HEIGHT / 2 : IMAGE_HEIGHT}
-            borderRadius="$m"
+            borderRadius="$s"
           />
         )}
         {post.title && (
           <Text
+            fontWeight={'500'}
```
Comment (patosullivan) [inline packages/ui/src/components/NotebookPost/NotebookPost.tsx:72]: Should use one of the tamagui tokens here, and if we don't have one maybe we should add one.

**Review by patosullivan — APPROVED** [review]
Looks good, just a few nits

File: packages/shared/src/logic/utils.ts (hunk 1 above)
Comment (jamesacklin) [reply packages/shared/src/logic/utils.ts:124]: Absolutely nothing. Thanks

**Review by patosullivan — APPROVED** [review]
Looks great!

**Pull request merged**